src/main/scala-2.12/org/apache/james/gatling/jmap/JmapMessages.scala

```diff
@@ -150,7 +152,13 @@ object JmapMessages {
   val listMessagesChecks: Seq[HttpCheck] = List(
       status.is(200),
       JmapChecks.noError,
-      jsonPath("$[0][1].messageIds[*]").findAll.saveAs("messageIds"))
```

Comment:
:+1: 

Maybe we could add a seperate check for 'non emptyness' that existing scenario could compose in order not to change their behaviour?